dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
+  -# A teacher is trying to view a student's progress on a level they have not started.
+  = render partial: 'levels/not_started'
```

Comment:
why would you need the layout? We are inside a page that has a layout already. If you do layout :true I would expect to get the layout rendered twice (or maybe it doesn't do anything for render :text?).

you could just say `= t(:not_started)` if you don't want a partial -- presumably the partial is there for future expansion of the page. (you don't need I18n in views because I18n is included)